atom/browser/api/atom_api_app.cc

```diff
@@ -1095,7 +1126,8 @@ void App::BuildPrototype(
       .SetMethod("getFileIcon", &App::GetFileIcon)
       .SetMethod("getAppMetrics", &App::GetAppMetrics)
       // TODO(juturu): Remove in 2.0, deprecate before then with warnings
-      .SetMethod("getAppMemoryInfo", &App::GetAppMetrics);
+      .SetMethod("getAppMemoryInfo", &App::GetAppMetrics)
+      .SetMethod("getGPUFeatureStatus", &App::GetGPUFeatureStatus);
```

Comment:
Mind moving this above the `//  TODO` comment so we don't think it is marked for deprecation like `getAppMemoryInfo` is?